File: src/BukkitPE/event/inventory/InventoryEvent.php
<?php
/**
 * Inventory related events
 */
namespace BukkitPE\event\inventory;

use BukkitPE\event\Event;
use BukkitPE\inventory\Inventory;

abstract class InventoryEvent extends Event{

	/** @var Inventory */
	protected $inventory;

	public function __construct(Inventory $inventory){
		$this->inventory = $inventory;
	}

	/**
	 * @return Inventory
	 */
	public function getInventory(){
		return $this->inventory;
	}

	/**
	 * @return \BukkitPE\entity\Human[]
	 */
	public function getViewers(){
		return $this->inventory->getViewers();
	}
}

File: src/net/BukkitPE/event/inventory/InventoryEvent.java
package net.BukkitPE.event.inventory;

import net.BukkitPE.Player;
import net.BukkitPE.event.Event;
import net.BukkitPE.inventory.Inventory;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public abstract class InventoryEvent extends Event {

    protected Inventory inventory;

    public InventoryEvent(Inventory inventory) {
        this.inventory = inventory;
    }

    public Inventory getInventory() {
        return inventory;
    }

    public Player[] getViewers() {
        return this.inventory.getViewers().stream().toArray(Player[]::new);
    }

}
